README.md

```diff
@@ -182,19 +182,110 @@ git checkout main
+- spotWorld:  Spot's world frame (defined by where spot's home is reset). The local_launch shell script publishes a static tf of spotWorld wrt map
+- spot:       Spot's base frame (defined by current location of spot). This frame is published by Proprioception node in local_launch shell script
+- marker:     Marker frame (defined by the center of spot dock's april tag). This frame is published by Episodic_memory_robotic_fetch node at the start of the node when spot looks at the marker and detects it
+- ariaWorld:  Aria's world frame (defined by location where Aria's SLAM pipeline runs). This frame is published by AriaLiveReader node
+- aria:       Aria's cpf frame (defined by location of Aria's camera). This frame is published by AriaLiveReader node everytime it receives a pose from Aria.
+
+Extra Aria frames (for information only):
+- cpf: Center frame of Aria with Z pointing out, X pointing up and Y pointing left
+- device: Base frame of Aria which is aligned with left-slam camera frame
+
+
+### Run ARIA - Spot Fetch tasks from LIVE Aria Stream
+- This mode requires a Multi - PC setup.
+    - PC #1 can be any Linux workstation that is capable to run low-level spot skills
+    - PC #2 should be a linux laptop (with good compute) which **will run the live streamer code in your bagpack while connected to Aria device via USB**
+
+#### Steps for PC #1
+ - Run Steps 1-4 from [above](#running-the-demo-asclscseq-experts)
+
+#### Step5. Run RVIZ visualization
+- Open a new terminal and cd into root of spot-sim2real repo
+- **Only if you have native ROS Noetic installation run the following steps to activate it**. Native ROS' RVIZ seems to behave better than the one inside virtual environment (from robostack).
+    ```bash
+    mamba deactivate
+    source /opt/ros/noetic/setup.bash
+    ```
+- Run RVIZ for visualization along with the relevant .rviz config file
+    ```bash
+    rosrun rviz rviz -d ./aria_data_loaders/aria_data_utils/rviz/spot-aria.rviz
+    ```
+
+#### Step6. Run the Episodic Memory Robotic Fetch demo node
+- Make sure robot is docked
+- In a new terminal
+    ```bash
+    cd aria_data_loaders/aria_data_utils/
+    python episodic_memory_robotic_fetch.py --verbose=True --use-policies=False
+    ```
+- Spot will standup from dock & navigate to the left of the dock & sit down so as to view the marker.
+- After you see `spot`, `spotWorld` and `marker` frames in RVIZ, you need to get Aria in the same frame system.
+- Run the steps for PC#2 as described [below](#steps-for-pc-2) to enable Aria's streaming
+- Once Aria's streaming has started and you are wearing Aria while having the PC#2 (i.e. laptop) in your bagback, **move to exactly where the spot is sitting and view the QR code of spot's dock for 2 seconds**. Make sure you can also see `aria` & `ariaWorld` frames in the same RVIZ window and the odometry of aria constantly updating as you are moving.
```

Comment:
@akshararai 
It doesn't need to be where spot is sitting. It should just be within 2m range of marker (as QR detection was accurate there).

User would not know where is the 2m radius around marker so it would be easier for user to go near where spot is ... because we make spot to go and sit at a hard-defined spot to view the marker. This is just to make things easy for user.

But I have removed this requirement from readme now.